taskapp/src/App.css

```diff
@@ -1,21 +1,89 @@
 .App {
-  text-align: center;
+	text-align: left;
+	background-color: #93d9f6;
 }
 
 .App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+  animation: App-logo-spin infinite 10s linear;
+	height: 40px;
+	float: left;
 }
 
 .App-header {
-  background-color: #222;
-  height: 150px;
+  background-color:#34a4da;
+  /* height: 100px; */
   padding: 20px;
-  color: white;
+  color: #000;
 }
+.App-header nav ul li a{
```

Comment:
This is not a good approach for CSS, try create component variations so that they can be reused